src/app/cube/details/action-panel/action-panel.component.html

```diff
@@ -19,6 +19,19 @@
         <i class="fal fa-spinner-third fa-spin"></i>
       </span>
     </button>
+
+    <span class="btn-group vertical" *ngIf="isViewable">
+      <button class="button neutral on-white" id="edit-button" type="submit"> 
+        <a (click)="openAdminApp()">Revise</a>
+
+      </button>
+    </span>
+    <!--
```

Comment:
We shouldn't leave commented out code lingering in the code base. If it isn't being used we should delete it to reduce clutter.